packages/fetch-error-handler/lib/create-handler.js

```diff
@@ -25,7 +25,7 @@ const { Writable } = require('node:stream');
  *     The response HTTP status code.
  * @property {string} url
  *     The URL of the response.
- * @property {FetchResponseBody} body
+ * @property {NodeFetchResponseBody | ReadableStream<Uint8Array> | null} body
```

Comment:
`ReadableStream` here is the native web streams type, which comes from the DOM lib types. i'm not 100% certain how this will behave if the consumer doesn't have that lib.

Reply:
actually this seems to work fine in cp-content-pipeline-schema